src/reducers/reducers.js

```diff
@@ -8,6 +8,13 @@ import {
   VisibilityReducer as visibility,
 } from 'ovirt-ui-components'
```

Comment:
when moving from a single file `foo.js` to a directory `foo` with mutliple files I would prefer the main file to be called `index.js`. This will allow other code that imported `foo` to work as before and also enforces standardized naming across the modules.